package.json

```diff
@@ -153,9 +153,8 @@
     "@svgr/webpack": "^8.0.1",
     "@swc/core": "^1.3.99",
     "@testing-library/cypress": "^9.0.0",
-    "@testing-library/dom": "^7.29.0",
```

Comment:
Nice! It's very possible this should allow us to revert https://github.com/metabase/metabase/pull/40084
(see https://github.com/testing-library/user-event/discussions/906#discussioncomment-2522468)